ddtrace/tracer/tracer.go

```diff
@@ -384,6 +384,7 @@ func (t *tracer) flush() {
 // pushPayload pushes the trace onto the payload. If the payload becomes
 // larger than the threshold as a result, it sends a flush request.
 func (t *tracer) pushPayload(trace []*span) {
+	t.config.statsd.Histogram("datadog.tracer.trace_span_count", float64(len(trace)), nil, 1)
```

Comment:
Would `trace_size` be better?

Reply:
I think `trace_size` is too ambiguous, but I'm open to better names.